addons/web/static/src/scss/bootstrap_review.scss

```diff
@@ -80,6 +80,21 @@
     }
 }
 
+// Do not display the backdrop element added by bootstrap in the body and added a
+// background on the modal to keep the same effect. The bootstrap backdrop was useful
+// for compatibility with <IE9 but is no longer needed. This also avoids z-index
+// issues because the wrapwrap (z-index: 0) can contain a modal. And in this case,
+// the backdrop (z-index: 999) goes above the modal.
+.modal-backdrop {
+    display: none;
+}
+.modal:not(.o_modal_no_backdrop) {
+    background-color: rgba($modal-backdrop-bg, $modal-backdrop-opacity);
+}
+.modal[data-backdrop="false"] {
+    background: 0;
```

Comment:
background: 0?

Reply:
Yes, you can use 0 on the background property as it will be interpreted as the background-position and the other background sub-properties will be reset to initial state so what you wanted to achieve = no more background color will be done... I can't find this to be a good practice though but fine, if you want to keep it that hard I won't cry over it :)